core/src/test/java/org/elasticsearch/index/mapper/date/SimpleDateMappingTests.java

```diff
@@ -174,23 +190,6 @@ private void assertNumericTokensEqual(ParsedDocument doc, DocumentMapper default
         assertThat(pos, equalTo(values.size()));
     }
 
-    public void testTimestampAsDate() throws Exception {
```

Comment:
Why was this test removed? Shouldn't this still work since epoch_millis is part of the default format?